modules/exploits/linux/smtp/exim_gethostbyname_bof.rb

```diff
@@ -3,51 +3,62 @@
+        'Name' => 'Exim GHOST (glibc gethostbyname) Buffer Overflow',
+        'Description' => %q{
+          This module remotely exploits CVE-2015-0235, aka GHOST, a heap-based
+          buffer overflow in the GNU C Library's gethostbyname functions on x86
+          and x86_64 GNU/Linux systems that run the Exim mail server.
+        },
+        'Author' => [
           'Unknown', # Discovered and published by Qualys, Inc.
         ],
-      'License' => BSD_LICENSE,
-      'References' => [
-        ['CVE', '2015-0235'],
-        ['US-CERT-VU', '967332'],
-        ['OSVDB', '117579'],
-        ['BID', '72325'],
-        ['URL', 'https://www.qualys.com/research/security-advisories/GHOST-CVE-2015-0235.txt'],
-        ['URL', 'https://community.qualys.com/blogs/laws-of-vulnerabilities/2015/01/27/the-ghost-vulnerability'],
-        ['URL', 'http://r-7.co/1CAnMc0'] # MSF Wiki doc (this module's manual)
-      ],
-      'DisclosureDate' => '2015-01-27',
-      'Privileged' => false, # uid=101(Debian-exim) gid=103(Debian-exim) groups=103(Debian-exim)
-      'Platform' => 'unix', # actually 'linux', but we execute a unix-command payload
-      'Arch' => ARCH_CMD, # actually [ARCH_X86, ARCH_X64], but ^
-      'Payload' => {
-        'Space' => 255, # the shorter the payload, the higher the probability of code execution
-        'BadChars' => "", # we encode the payload ourselves, because ^
-        'DisableNops' => true,
-        'ActiveTimeout' => 24*60*60 # we may need more than 150 s to execute our bind-shell
-      },
-      'Notes' => {'AKA' => ['ghost']},
-      'Targets' => [['Automatic', {}]],
-      'DefaultTarget' => 0
-    ))
+        'License' => BSD_LICENSE,
+        'References' => [
+          ['CVE', '2015-0235'],
+          ['US-CERT-VU', '967332'],
+          ['OSVDB', '117579'],
+          ['BID', '72325'],
+          ['URL', 'https://www.qualys.com/research/security-advisories/GHOST-CVE-2015-0235.txt'],
+          ['URL', 'https://community.qualys.com/blogs/laws-of-vulnerabilities/2015/01/27/the-ghost-vulnerability'],
+          ['URL', 'http://r-7.co/1CAnMc0'] # MSF Wiki doc (this module's manual)
+        ],
+        'DisclosureDate' => '2015-01-27',
+        'Privileged' => false, # uid=101(Debian-exim) gid=103(Debian-exim) groups=103(Debian-exim)
+        'Platform' => 'unix', # actually 'linux', but we execute a unix-command payload
+        'Arch' => ARCH_CMD, # actually [ARCH_X86, ARCH_X64], but ^
+        'Payload' => {
+          'Space' => 255, # the shorter the payload, the higher the probability of code execution
+          'BadChars' => '', # we encode the payload ourselves, because ^
+          'DisableNops' => true,
+          'ActiveTimeout' => 24 * 60 * 60 # we may need more than 150 s to execute our bind-shell
+        },
+        'Notes' => {
+          'AKA' => ['ghost'],
+          'Stability' => [CRASH_SERVICE_DOWN],
+          'SideEffects' => [IOC_IN_LOGS],
+          'Reliability' => [REPEATABLE_SESSION]
```

Comment:
Not a blocker; `CRASH_SERVICE_DOWN` and `REPEATABLE_SESSION` seem mutually exclusive 🤔